tests/app/clients/test_aws_pinpoint.py

```diff
@@ -170,3 +170,30 @@ def test_send_sms_returns_result_with_non_retryable_error_delivery_status(
     aws_pinpoint_client.statsd_client.incr.assert_called_with(
         f'clients.pinpoint.delivery-status.{delivery_status.lower()}'
     )
+
+
+def test_send_sms_raises_invalid_provider_error_with_invalide_number(aws_pinpoint_client, boto_mock):
+    delivery_status = 'PERMANENT_FAILURE'
+    invalid_number = '+12223334444'
```

Comment:
This is only used once below, which means you don't strictly need a variable.  I don't insist on changing it.

Reply:
True, but I like labeling it as an invalid number